src/stage1/codegen.cpp

```diff
@@ -9414,7 +9459,7 @@ static void add_fp_entry(CodeGen *g, const char *name, uint32_t bit_count, LLVMT
 {
     ZigType *entry = new_type_table_entry(ZigTypeIdFloat);
     entry->llvm_type = type_ref;
-    entry->size_in_bits = 8*LLVMStoreSizeOfType(g->target_data_ref, entry->llvm_type);
```

Comment:
@andrewrk this is how I was planning to solve https://github.com/ziglang/zig/issues/11587